numpy/core/tests/test_umath.py

```diff
@@ -17,6 +18,15 @@
     _gen_alignment_data, assert_array_almost_equal_nulp, assert_warns
     )
 
+def get_glibc_version():
+    ver = 0.0
+    try:
+        ver = float(os.confstr('CS_GNU_LIBC_VERSION').rsplit(' ')[1])
+    except Exception as inst:
+        print(inst)
+
+    return ver
```

Comment:
I think this "hammer" is too big - now those tests would also be skipped on non-glibc systems (mainly windows).

Also, I'm not sure if it's a good idea to pretend a version number is a float - python can do comparisons on strings just fine.

Reply:
Maybe pass the needed gcc version and return `True` (run the test) or `False` (skip the test), with non-gcc platforms returning `True`. I also wonder about clang (FreeBSD, Mac). Or maybe just mark the test as xfail for now.

Reply:
I used a conditional xfail. Will work on windows too.